### ISSUE TITLE
TextField Accessiblity - validateOnFocusOut errors are not read by narrator

### ISSUE THREAD

**Issue by salehaHussain** [OfficeDev/office-ui-fabric-react#3418]
### Bug Report

- __Package version(s)__: 5.8.0
- __Browser and OS versions__: Edge Narrator combo

#### Priorities and help requested

Are you willing to submit a PR to fix? No

Requested priority: Normal

#### Describe the issue:

On using Narrator and Edge, setting the error properties fro textfield as validateOnLoad: false, validateOnFocusOut: true, the narrator does no read out the error on tabbing out of the textfield, even though it appears on the UI.

#### Expected behavior:

The error should be read aloud immediately, and not when we focus back on the textArea,

**Comment by mdahamiwal** [comment]
This happens only for Edge and I'm following up with Edge team for this issue. I suspect the error announcement is subside due to focus on next active element which shift the announcement priority. I tested for firefox + NVDA and it works fine.

**Comment by salehaHussain** [comment]
Yes, Edge Narrator is the real problem. NVDA is working perfectly.

**Comment by aditima** [comment]
@mdahamiwal - are you planning to work on a fix for this?

**Comment by michaelangotti** [comment]
@salehaHussain @mdahamiwal There hasn't been any momentum on this one. Is this still needed? Was the Edge team able to provide additional info/fix?

**Comment by mdahamiwal** [comment]
Sorry for the delayed response here. With RS4 windows release there have been improvements in Edge for focus out announcements. We confirmed with the internal a11y testing team and they did sign-off on the current experience. I am closing the issue, please feel free to re-activate if required.
